actions/workflows/st2_pkg_e2e_test.yaml

```diff
@@ -117,7 +132,10 @@ tasks:
     action: core.remote_sudo
     input:
       hosts: <% ctx().vm_info.private_ip_address %>
+      # If pipe_into_bootstrap_script is not empty, return it + " | "
+      # If pipe_into_bootstrap_script is empty, return an empty string
       cmd: >-
+        <% switch(ctx().pipe_into_bootstrap_script => concat(ctx().pipe_into_bootstrap_script, " | "), true => "") %>
```

Comment:
That should work in this particular case.

However I provided `--u16-add-insecure-py3-ppa` flag as identified more bash corner cases with the interactive mode under different environments. https://github.com/StackStorm/st2-packages/pull/682